homeassistant/components/comelit/climate.py

```diff
@@ -119,10 +119,10 @@ def __init__(
         # because no serial number or mac is available
         self._attr_unique_id = f"{config_entry_entry_id}-{device.index}"
         self._attr_device_info = coordinator.platform_device_info(device, device.type)
+        self._update_attributes()
 
-    @callback
-    def _handle_coordinator_update(self) -> None:
-        """Handle updated data from the coordinator."""
+    def _update_attributes(self) -> None:
```

Comment:
```suggestion
    @callback
    def _update_attributes(self) -> None:
```
I think this still needs to be marked as `@callback` :thinking:

Reply:
this also counts for the humidity entity

Reply:
after reading https://developers.home-assistant.io/docs/asyncio_categorizing_functions i think it falls under "Event loop and thread safe" and does not need to be annotated